Userland/Libraries/LibGUI/TextDocument.cpp

```diff
@@ -884,6 +884,7 @@ void RemoveTextCommand::undo()
     m_document.set_all_cursors(new_cursor);
 }
 
+<<<<<<< HEAD
```

Comment:
This shouldn't be here. :^) I see you removed it in the second commit, but please remove it in this commit instead.